.github/codeql/codeql-config.yml

```
@@ -0,0 +1,23 @@
+  - "backend/venv/**"
+  - "**/*.test.js"
+  - "**/*.test.ts"
+  - "**/*.test.tsx"
+  - "**/test/**"
+  - "**/tests/**"
+  - "**/__tests__/**"
+
+paths:
+  - "frontend/src/**"
+  - "backend/**"
+  - "!backend/venv/**"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)